install.sh

```diff
@@ -57,7 +57,7 @@ __hoard_install_mac(){
     echo "brew tap add Hyde46/hoard"
     echo "brew install hoard"
     echo "To install as zshell plug, run:"
-    echo 'curl https://raw.githubusercontent.com/Hyde46/hoard/main/src/shell/hoard.zsh >> ~/.zshrc'
+    echo 'echo "hoard shell_config zsh | source" >> ~/.zshrc'
```

Comment:
I think i'd prefer evaluating the hoard command and append that to the shell rc's for zsh same for bash
```
eval "$(hoard shell_config zsh)" >> ~/.zshrc
```

Reply:
Ah gotcha, I see the point now 👍